### DISCUSSION TITLE
--this developer provides no support--

### DISCUSSION THREAD

**Discussion by runthis** [spatie/laravel-medialibrary#2546] [General] [answered]
Just a heads up to future readers who click the Closed tab (since this will be closed). They just close bug reports and move them to "Discussion" which no one has replied to since February (6 months).

If I had a chance to properly try what they made, I probably would have purchased their "pro" version. But immediately closing my issue after 30 seconds and sending me off to a queue that no one responds to in 6 months for something I tried to resolve on my own for several hours is weak. Super weak.

🥱🙄

**Comment by freekmurze** [comment, marked as answer]
I'm sorry we don't meet your standards. We put a lot of love in this package and do appreciate any feedback and PRs that make it better.

However, we don't have the bandwidth to answer every question on the basic functionality of this package. This package is working well for us (and thousands of other users). 

If you feel our docs could be improved, then experiment with the package, or take a look at how it works (possibly via the tests or the videos we recorded https://spatie.be/videos/discovering-laravel-media-library). Feel free to submit a PR to improve the docs in any way you should see fit.

I do not appreciate negative, entitled comments. Here are some resources for you to check out:

https://spatie.be/guidelines/open-source#handling-feedback
https://gist.github.com/g1eny0ung/9e7d4d0f72547a8d156452e76fa8f7a3
https://github.com/alpinejs/alpine/discussions/1423

**Reply by runthis** [reply]
Feels like instead of writing the 3 paragraphs, probably could have helped with my likely basic question you knew the answer to. But no worries.

**Reply by freekmurze** [reply]
I agree that I could have done better things with my time.

**Reply by runthis** [reply]
I am sorry for any offense. I did take the time to dig into your project. I didn't just come here with nothing to share. I spent time. I listed everything I tried and I did the responsible thing and reported a bug (of which I eventually found the solution that is related to a bug in this repo, or a lack of documentation, or both). 

The bug was disregarded immediately. You don't have to provide support, absolutely, but if you want people to help you by asking them to make a PR, helping them doesn't seem too out of the question. To me it seems weird and one-sided, and is not "entitled" to let other developers know that this package does not provide support, which is the actual single thing businesses look at.

imho, this package has an upsell for a paid version and I feel some empathy for your end users would be a way to sell more paid versions. I looked in the repositories of some users who have had their reports moved to the Discussions section and lined up probably which repo they were working on at the time and they weren't using the product anymore. But showing a little empathy could have turned them into lifelong customers and introduced them to the rest of the products being sold.

Anyways I am probably wasting my time saying any of this I guess. Like you said, "This package is working well for us" so if you aren't experiencing any issues, it's probably fine. The old "Works for me" reply 😄

As for the original issue. You tell people to do: `$newsItem = News::find(1);` -- Which after a fresh install with no items in the table after migration, that is a null reference (which means they will get an error). If they instead just pass in `new News()` or use Laravels zero setup to reference the model in the controller, it won't go to the database or upload the file (but still report success, falsely, as there appears to be no check that it did either thing). So that is the bug I experienced as a user of your package and that is how to replicate it. I am not going to open a PR for this, feel free to fix it, or don't. I am sorry end users are such an inconvenience to your project.

Wish you luck!
